mivs/automated_emails.py

```diff
@@ -65,7 +65,8 @@ def __init__(self, *args, **kwargs):
           ident='mivs_game_waitlisted')
 
 MIVSEmail(IndieGame, 'Last chance to accept your MIVS booth', 'game_accept_reminder.txt',
-          lambda game: game.status == c.ACCEPTED and not game.confirmed, when=days_before(2, c.MIVS_CONFIRM_DEADLINE),
+          lambda game: game.status == c.ACCEPTED and not game.confirmed
+                       and (localized_now() - timedelta(days=2)) > game.studio.confirm_deadline,
```

Comment:
Now that the deadline is a property of the game's studio, we can't pass it to the `when` parameter anymore, so we unfortunately have to include the date-based filter manually.